client/app/hosting/translations/Messages_fr_FR.xml

```diff
@@ -1528,7 +1528,14 @@
    <translation id="common_subscription_contacts_admin">Administrateur</translation>
    <translation id="common_subscription_contacts_billing">Facturation</translation>
    <translation id="common_subscription_contacts_tech">Technique</translation>
-   <translation id="core_service_expiration_date">Date d'expiration</translation>
    <translation id="core_service_expiration_auto_renew">Renouvellement automatique prévu en</translation>
 
+   <translation id="expiration_new_tab">S'ouvre dans un nouvel onglet</translation>
+   <translation id="expiration_renew_date">Renouvellement automatique prévu en </translation>
```

Comment:
Could you please add the parameters directly in the translation ? :)

`<translation id="expiration_renew_date">Renouvellement automatique prévu en {{renewalDate}}</translation>`